code/modules/modular_computers/obj/item/role_tablet_presets.dm

```diff
@@ -289,7 +295,7 @@
 
 /obj/item/modular_computer/pda/lawyer/Initialize(mapload)
 	. = ..()
-	for(var/datum/computer_file/program/messenger/messenger_app in stored_files)
+	for(var/datum/computer_file/program/messenger/messenger_app in cpu.stored_files)
```

Comment:
Just as an example, I had removed computer parts so we wouldn't have this weirdness "check for part then check stored files on that", and this just feels like going back to that but altered.

Reply:
ditto